packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -32,20 +54,13 @@ const nextConfig = {
 
   // use this configuration to ensure that only images from the whitelisted domains
   // can be served from the Next.js Image Optimization API
+  // uses remotePatterns presets out of the box
   // see https://nextjs.org/docs/app/api-reference/components/image#remotepatterns
   images: {
-    remotePatterns: [
-      {
-        protocol: 'https',
-        hostname: 'edge*.**',
-        port: '',
-      },
-      {
-        protocol: 'https',
-        hostname: 'feaas*.blob.core.windows.net',
-        port: '',
-      },
-    ],
+    remotePatterns: 
+      process.env.NODE_ENV === 'development' || process.env.SITECORE ? 
```

Comment:
I don't think we should include `process.env.SITECORE` (editing mode) here. For one, they could be using a non-XMC editing host (without this var); but mainly, this has the potential to cause issues when publishing (i.e. my image worked when building the page, but broken on prod after publish).

I think authors should be using the same configuration as production.

Reply:
If editing uses the production preset then we'd have the reverse situation of authors being confused about feaas images not working. I think having it enabled for editing would lead to better author-dev workflow. Instead of pinging devs to add a source each time authors add an image, they can ping them once when publishing, with a list of sources to add.

Reply:
Surprises at authoring time are always preferred to surprises on publish/live site. Authors pinging devs when drafting new content is preferred than when content is broken on the live site at which point it's an emergency.

Isn't breaking existing images in FEAAS components a concern for live mode as well (not just editing)?

We have to be careful here because this does potentially have [cost associations](https://vercel.com/docs/image-optimization/managing-image-optimization-costs). I'm starting to wonder if this should even be on by default (for FEAAS images).